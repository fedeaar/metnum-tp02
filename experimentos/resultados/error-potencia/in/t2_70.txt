9.395234000000000000e+06 3.959047000000000000e+06 -1.026156000000000000e+06 6.160800000000000000e+05 -9.731010000000000000e+05 -2.334450000000000000e+05 2.639100000000000000e+05 1.137764000000000000e+06 9.194890000000000000e+05 -6.944570000000000000e+05 -1.171346000000000000e+06 7.764060000000000000e+05 -1.199786000000000000e+06 -2.468244000000000000e+06 2.425677000000000000e+06 2.493320000000000000e+06 1.637228000000000000e+06 1.440478000000000000e+06 2.133400000000000000e+05 1.452751000000000000e+06
3.959047000000000000e+06 4.491325000000000000e+06 -2.128989000000000000e+06 2.088000000000000000e+06 -1.809434000000000000e+06 7.703160000000000000e+05 2.834700000000000000e+04 1.795162000000000000e+06 5.240200000000000000e+05 6.684980000000000000e+05 -1.393987000000000000e+06 3.057010000000000000e+05 7.530500000000000000e+05 -6.495280000000000000e+05 1.575257000000000000e+06 9.395710000000000000e+05 9.533350000000000000e+05 1.882371000000000000e+06 7.986700000000000000e+04 -6.621640000000000000e+05
-1.026156000000000000e+06 -2.128989000000000000e+06 5.325922000000000000e+06 -4.891470000000000000e+05 2.374041000000000000e+06 -1.368491000000000000e+06 -1.060102000000000000e+06 -7.487750000000000000e+05 5.177580000000000000e+05 -1.930634000000000000e+06 2.550472000000000000e+06 -2.806605000000000000e+06 9.559370000000000000e+05 1.716394000000000000e+06 8.778460000000000000e+05 1.606430000000000000e+05 1.342410000000000000e+05 4.830250000000000000e+05 -1.653849000000000000e+06 3.411371000000000000e+06
6.160800000000000000e+05 2.088000000000000000e+06 -4.891470000000000000e+05 6.719169000000000000e+06 -8.847420000000000000e+05 -8.598540000000000000e+05 -7.467840000000000000e+05 -8.641800000000000000e+04 1.278206000000000000e+06 1.718681000000000000e+06 2.482754000000000000e+06 -2.235755000000000000e+06 1.214575000000000000e+06 -1.746230000000000000e+05 1.055045000000000000e+06 -8.969380000000000000e+05 2.030070000000000000e+05 2.702021000000000000e+06 -1.448899000000000000e+06 -2.472570000000000000e+05
-9.731010000000000000e+05 -1.809434000000000000e+06 2.374041000000000000e+06 -8.847420000000000000e+05 7.965140000000000000e+06 -3.157992000000000000e+06 -1.199100000000000000e+06 -2.573596000000000000e+06 2.734180000000000000e+05 -7.435090000000000000e+05 2.665981000000000000e+06 -4.808660000000000000e+05 -1.290602000000000000e+06 1.758685000000000000e+06 -2.343564000000000000e+06 2.185382000000000000e+06 -1.745255000000000000e+06 1.877164000000000000e+06 -2.106426000000000000e+06 6.241960000000000000e+05
-2.334450000000000000e+05 7.703160000000000000e+05 -1.368491000000000000e+06 -8.598540000000000000e+05 -3.157992000000000000e+06 8.140052000000000000e+06 2.963690000000000000e+05 6.485010000000000000e+05 -1.971094000000000000e+06 1.038655000000000000e+06 -3.664780000000000000e+05 7.094170000000000000e+05 -1.577620000000000000e+05 -2.106759000000000000e+06 -2.171740000000000000e+06 4.122570000000000000e+05 3.264370000000000000e+05 -3.285650000000000000e+05 8.237140000000000000e+05 -3.894940000000000000e+05
2.639100000000000000e+05 2.834700000000000000e+04 -1.060102000000000000e+06 -7.467840000000000000e+05 -1.199100000000000000e+06 2.963690000000000000e+05 9.924300000000000000e+06 1.479093000000000000e+06 3.091386000000000000e+06 -1.075309000000000000e+06 -4.646500000000000000e+04 1.402121000000000000e+06 -4.284590000000000000e+05 1.001115000000000000e+06 1.098293000000000000e+06 -2.255462000000000000e+06 -4.679280000000000000e+05 -9.667810000000000000e+05 -1.305027000000000000e+06 2.893314000000000000e+06
1.137764000000000000e+06 1.795162000000000000e+06 -7.487750000000000000e+05 -8.641800000000000000e+04 -2.573596000000000000e+06 6.485010000000000000e+05 1.479093000000000000e+06 8.596174000000000000e+06 7.580280000000000000e+05 -8.875320000000000000e+05 -2.061730000000000000e+05 1.913240000000000000e+06 3.832355000000000000e+06 -3.456300000000000000e+04 3.093659000000000000e+06 1.003363000000000000e+06 4.920787000000000000e+06 -2.568538000000000000e+06 4.145270000000000000e+05 -2.559905000000000000e+06
9.194890000000000000e+05 5.240200000000000000e+05 5.177580000000000000e+05 1.278206000000000000e+06 2.734180000000000000e+05 -1.971094000000000000e+06 3.091386000000000000e+06 7.580280000000000000e+05 4.547069000000000000e+06 9.525630000000000000e+05 2.176953000000000000e+06 3.758490000000000000e+05 -2.061590000000000000e+05 7.832110000000000000e+05 2.304194000000000000e+06 -7.441070000000000000e+05 1.715313000000000000e+06 1.699113000000000000e+06 -3.272730000000000000e+05 2.065826000000000000e+06
-6.944570000000000000e+05 6.684980000000000000e+05 -1.930634000000000000e+06 1.718681000000000000e+06 -7.435090000000000000e+05 1.038655000000000000e+06 -1.075309000000000000e+06 -8.875320000000000000e+05 9.525630000000000000e+05 9.773144000000000000e+06 1.422823000000000000e+06 -3.585670000000000000e+05 9.722540000000000000e+05 -1.995983000000000000e+06 1.254723000000000000e+06 3.177710000000000000e+05 2.553041000000000000e+06 1.674020000000000000e+06 -3.815090000000000000e+05 -1.433760000000000000e+06
-1.171346000000000000e+06 -1.393987000000000000e+06 2.550472000000000000e+06 2.482754000000000000e+06 2.665981000000000000e+06 -3.664780000000000000e+05 -4.646500000000000000e+04 -2.061730000000000000e+05 2.176953000000000000e+06 1.422823000000000000e+06 6.506137000000000000e+06 -1.494491000000000000e+06 6.833410000000000000e+05 7.790620000000000000e+05 3.658240000000000000e+05 6.305810000000000000e+05 1.278113000000000000e+06 2.112647000000000000e+06 -1.831048000000000000e+06 1.188785000000000000e+06
7.764060000000000000e+05 3.057010000000000000e+05 -2.806605000000000000e+06 -2.235755000000000000e+06 -4.808660000000000000e+05 7.094170000000000000e+05 1.402121000000000000e+06 1.913240000000000000e+06 3.758490000000000000e+05 -3.585670000000000000e+05 -1.494491000000000000e+06 5.446878000000000000e+06 2.914140000000000000e+05 -1.725898000000000000e+06 -6.190600000000000000e+05 9.702990000000000000e+05 8.405040000000000000e+05 -8.237560000000000000e+05 1.567387000000000000e+06 -7.473180000000000000e+05
-1.199786000000000000e+06 7.530500000000000000e+05 9.559370000000000000e+05 1.214575000000000000e+06 -1.290602000000000000e+06 -1.577620000000000000e+05 -4.284590000000000000e+05 3.832355000000000000e+06 -2.061590000000000000e+05 9.722540000000000000e+05 6.833410000000000000e+05 2.914140000000000000e+05 7.283116000000000000e+06 9.231980000000000000e+05 4.111911000000000000e+06 2.521239000000000000e+06 1.822108000000000000e+06 -1.280929000000000000e+06 -1.223521000000000000e+06 1.782945000000000000e+06
-2.468244000000000000e+06 -6.495280000000000000e+05 1.716394000000000000e+06 -1.746230000000000000e+05 1.758685000000000000e+06 -2.106759000000000000e+06 1.001115000000000000e+06 -3.456300000000000000e+04 7.832110000000000000e+05 -1.995983000000000000e+06 7.790620000000000000e+05 -1.725898000000000000e+06 9.231980000000000000e+05 5.292019000000000000e+06 -2.622900000000000000e+05 7.375480000000000000e+05 -6.943800000000000000e+04 -1.248827000000000000e+06 -1.533944000000000000e+06 -2.447900000000000000e+04
2.425677000000000000e+06 1.575257000000000000e+06 8.778460000000000000e+05 1.055045000000000000e+06 -2.343564000000000000e+06 -2.171740000000000000e+06 1.098293000000000000e+06 3.093659000000000000e+06 2.304194000000000000e+06 1.254723000000000000e+06 3.658240000000000000e+05 -6.190600000000000000e+05 4.111911000000000000e+06 -2.622900000000000000e+05 8.814027000000000000e+06 2.045871000000000000e+06 9.950530000000000000e+05 -8.876600000000000000e+05 -5.492400000000000000e+05 2.367400000000000000e+06
2.493320000000000000e+06 9.395710000000000000e+05 1.606430000000000000e+05 -8.969380000000000000e+05 2.185382000000000000e+06 4.122570000000000000e+05 -2.255462000000000000e+06 1.003363000000000000e+06 -7.441070000000000000e+05 3.177710000000000000e+05 6.305810000000000000e+05 9.702990000000000000e+05 2.521239000000000000e+06 7.375480000000000000e+05 2.045871000000000000e+06 8.002370000000000000e+06 6.323070000000000000e+05 5.428390000000000000e+05 -8.078580000000000000e+05 -2.314860000000000000e+05
1.637228000000000000e+06 9.533350000000000000e+05 1.342410000000000000e+05 2.030070000000000000e+05 -1.745255000000000000e+06 3.264370000000000000e+05 -4.679280000000000000e+05 4.920787000000000000e+06 1.715313000000000000e+06 2.553041000000000000e+06 1.278113000000000000e+06 8.405040000000000000e+05 1.822108000000000000e+06 -6.943800000000000000e+04 9.950530000000000000e+05 6.323070000000000000e+05 7.323965000000000000e+06 -3.719220000000000000e+05 3.036990000000000000e+05 -5.146730000000000000e+05
1.440478000000000000e+06 1.882371000000000000e+06 4.830250000000000000e+05 2.702021000000000000e+06 1.877164000000000000e+06 -3.285650000000000000e+05 -9.667810000000000000e+05 -2.568538000000000000e+06 1.699113000000000000e+06 1.674020000000000000e+06 2.112647000000000000e+06 -8.237560000000000000e+05 -1.280929000000000000e+06 -1.248827000000000000e+06 -8.876600000000000000e+05 5.428390000000000000e+05 -3.719220000000000000e+05 8.629767000000000000e+06 -1.677217000000000000e+06 7.627380000000000000e+05
2.133400000000000000e+05 7.986700000000000000e+04 -1.653849000000000000e+06 -1.448899000000000000e+06 -2.106426000000000000e+06 8.237140000000000000e+05 -1.305027000000000000e+06 4.145270000000000000e+05 -3.272730000000000000e+05 -3.815090000000000000e+05 -1.831048000000000000e+06 1.567387000000000000e+06 -1.223521000000000000e+06 -1.533944000000000000e+06 -5.492400000000000000e+05 -8.078580000000000000e+05 3.036990000000000000e+05 -1.677217000000000000e+06 5.271336000000000000e+06 -1.776463000000000000e+06
1.452751000000000000e+06 -6.621640000000000000e+05 3.411371000000000000e+06 -2.472570000000000000e+05 6.241960000000000000e+05 -3.894940000000000000e+05 2.893314000000000000e+06 -2.559905000000000000e+06 2.065826000000000000e+06 -1.433760000000000000e+06 1.188785000000000000e+06 -7.473180000000000000e+05 1.782945000000000000e+06 -2.447900000000000000e+04 2.367400000000000000e+06 -2.314860000000000000e+05 -5.146730000000000000e+05 7.627380000000000000e+05 -1.776463000000000000e+06 9.665004000000000000e+06
