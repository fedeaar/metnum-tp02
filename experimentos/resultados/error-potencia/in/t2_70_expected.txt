2.300818573074738681e+07
2.082385787309509143e+07
1.682454928028111905e+07
1.480802983040448278e+07
1.317197708459708467e+07
1.030243856810500845e+07
9.551263058192564175e+06
8.214581225526419468e+06
7.618844157157228328e+06
5.286509716592115350e+06
4.473089244701411575e+06
3.856457620830965228e+06
3.603361219495526981e+06
1.991342116111257579e+06
1.454814112511440879e+06
9.994051445169735234e+05
6.226809563126718858e+05
2.711321827761787572e+05
2.265492197441520693e+05
3.079658301056853816e+03
